CODEOWNERS

```
@@ -1,80 +1,65 @@
+src/handlers/fedora.ts @estebanmino
 
 # Model
-src/model/backup.ts @brunobar79 @estebanmino
-src/model/keychain.ts @brunobar79 @estebanmino
-src/model/migrations.ts @brunobar79 @estebanmino
-src/model/wallet.ts @brunobar79 @estebanmino
+src/model/backup.ts @brunobar79 @estebanmino @salman-rb @jkadamczyk
+src/model/keychain.ts @brunobar79 @estebanmino @salman-rb @estrattonbailey @skylarbarrera
+src/model/migrations.ts @brunobar79 @estebanmino @salman-rb @estrattonbailey @skylarbarrera
+src/model/wallet.ts @brunobar79 @estebanmino @salman-rb @estrattonbailey @skylarbarrera
 
 # redux
-src/redux/gas.ts @brunobar79 @estebanmino
-src/redux/nonceManager.ts @derHowie @brunobar79 @estebanmino
+src/redux/gas.ts @estebanmino @skylarbarrera @estrattonbailey
+src/redux/nonceManager.ts @derHowie @skylarbarrera @estrattonbailey
 
 # navigation
-src/navigation @osdnk @terrysahaidak
-
-# APP INFRA
-.eslintignore @brunobar79 @osdnk @estebanmino @estrattonbailey
-.eslintrc.js @brunobar79 @osdnk @estebanmino @estrattonbailey
-.gitattributes @brunobar79 @estebanmino
-.gitignore @brunobar79 @estebanmino
-.nvmrc @brunobar79 @osdnk @estebanmino
-.prettierrc.js @brunobar79 @osdnk @estebanmino
-.watchmanconfig @brunobar79 @osdnk @estebanmino
-
-# SECURITY / APP INFRA
-.yarnrc @brunobar79 @estebanmino
+src/navigation @jkadamczyk @skylarbarrera @estrattonbailey
 
 # APP INFRA
-app.json @brunobar79 @estebanmino
-
-# SECURITY / APP INFRA
-audit-ci.json @brunobar79 @skylarbarrera @estebanmino @nickbytes
-
-# APP INFRA
-babel.config.js @brunobar79 @osdnk @estebanmino
-CODEOWNERS @brunobar79 @estebanmino @nickbytes
-globals.d.ts @brunobar79 @osdnk @estebanmino
-globalVariables.js @brunobar79 @osdnk @estebanmino
-
-# HARDHAT
-hardhat.config.js @brunobar79 @estebanmino
-
-# APP CORE
-index.js @brunobar79 @estebanmino @nickbytes
-
-# LEGAL
-LICENSE @jinchung @brunobar79 @nickbytes @estebanmino
-
-# SECURITY / APP INFRA
-metro.config.js @brunobar79 @osdnk @terrysahaidak @estebanmino
-metro.transform.js @brunobar79 @osdnk @estebanmino
-package.json @brunobar79 @estebanmino @nickbytes
+package.json @salman-rb @skylarbarrera @estrattonbailey
+.eslintignore @salman-rb @estrattonbailey @skylarbarrera
+.eslintrc.js @salman-rb @estrattonbailey @skylarbarrera
+.gitattributes @brunobar79 @salman-rb @estrattonbailey @skylarbarrera
+.gitignore @salman-rb @estrattonbailey @skylarbarrera
+.nvmrc @salman-rb @estrattonbailey @skylarbarrera
+.prettierrc.js @salman-rb @estrattonbailey @skylarbarrera
+.watchmanconfig @jkadamczyk
+.yarnrc @estrattonbailey @skylarbarrera
+audit-ci.json @brunobar79 @salman-rb @skylarbarrera @estebanmino @estrattonbailey
+babel.config.js @brunobar79 @estrattonbailey @jkadamczyk @skylarbarrera
+globals.d.ts @estrattonbailey @skylarbarrera
+yarn.lock @salman-rb @estrattonbailey @skylarbarrera
+tsconfig.json @salman-rb @estrattonbailey @skylarbarrera
+hardhat.config.js @salman-rb @skylarbarrera @estebanmino
```

```suggestion
hardhat.config.js @salman-rb @skylarbarrera
```

Removing Esteban, you can ping us if needed